docs/internal.md

```diff
@@ -109,6 +109,8 @@ The volume label is "cidata", as defined by [cloud-init NoCloud](https://cloudin
 - `LIMA_CIDATA_UID`: the numeric UID
 - `LIMA_CIDATA_MOUNTS`: the number of the Lima mounts
 - `LIMA_CIDATA_MOUNTS_%d_MOUNTPOINT`: the N-th mount point of Lima mounts (N=0, 1, ...)
+- `LIMA_CIDATA_MOUNTS_%d_MOUNTOPTION`: the N-th mount option of Lima mounts (N=0, 1, ...)
+- `LIMA_CIDATA_MOUNTTYPE`: the type of the Lima mounts ("sshfs", "9p", ...)
```

Comment:
```suggestion
- `LIMA_CIDATA_MOUNTTYPE`: the type of the Lima mounts ("reverse-sshfs", "9p", ...)
```

Reply:
Hmm, the mount itself (in the guest) is of type "sshfs" - even if it is "reverse sshfs" when seen from the host